stan/math/rev/core/var.hpp

```diff
@@ -290,7 +290,108 @@ class var_value {
    * @return The result of dividing this variable by the specified
    * variable.
    */
-  inline var_value<T>& operator/=(T b);
+  inline var_value<T, VariType>& operator/=(T b);
+
+  /**
+   * A block view of the underlying Eigen matrices.
+   * @param i Starting row of block.
+   * @param j Starting columns of block.
+   * @param p Number of rows to return.
+   * @param q Number of columns to return.
+   */
+  inline const auto block(Eigen::Index i, Eigen::Index j, Eigen::Index p,
```

Comment:
[optional] These view functions are very similar. I am starting to think about macros..

Reply:
I thought about that but I feel comfortable cargo-culting no macros here. The verbosity is pretty fine and idt this will be repeated in many other places